auditevent.go

```diff
@@ -113,6 +113,28 @@ func NewAuditEvent(
 	}
 }
 
+// NewAuditEvent returns a new AuditEvent with the passed AuditID.
+func NewAuditEventWithID(
```

Comment:
Let's add some simple test for this function too, so we get test coverage. Something as simple as creating the audit event object and making sure that it has the same ID as was given.